docs/docs/en/architecture/configuration.md

```diff
@@ -185,19 +185,21 @@ Currently, common.properties mainly configures Hadoop,s3a related configurations
 |Parameters | Default value| Description|
 |--|--|--|
 data.basedir.path|/tmp/dolphinscheduler| local directory used to store temp files
-resource.storage.type|NONE| type of resource files: HDFS, S3, NONE
-resource.upload.path|/dolphinscheduler| storage path of resource files
+resource.view.suffixs| txt,log,sh,conf,cfg,py,java,sql,hql,xml,properties| file types supported by resource center
```

Comment:
We have removed the suffix restriction from the content center in #9553. So please remove it.